### PR TITLE
Hide StatusBar Icons

### LINKED ISSUE #584: Status bar stuff persists when non-editor pane items are active
With the pre-1.0 version of linter, if the active pane item wasn't a `TextEditor`, then all of the linter-related status bar tiles were hidden - it would be great to see this added to post-1.0 linter. It can be quite noisy to see linter status when the active pane item can't be linted.

### PR DESCRIPTION
**Pull request by steelbrain**
Fixes #584
Not hiding the count as discussed. Only hiding project and file buttons.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Add set-bubble-transparent command to set the bubble transparent until the key is released (Fixes #608)
 * Deselect bottom tab when error panel is hidden
 * Add linter:lint command (Fixes #624)
+* Hide Status Bar buttons when Active Pane is not an editor
```
Comment (k2b6s9j) [inline CHANGELOG.md]: You're going to need to merge master back into this branch. All the "Upcoming" changes above this were released as `1.0.5`.

Comment (steelbrain) [inline CHANGELOG.md]: You should've atleast told me :smiley:

**Comment by k2b6s9j** [comment]
Merge master back in, and we'll be good.

**Comment by steelbrain** [comment]
@k2b6s9j done

**Comment by k2b6s9j** [comment]
Woot. :+1:

**Pull request merged**